Assets/Scripts/Utils/Tags.cs

```diff
@@ -3,4 +3,7 @@ public static class Tags
     public const string PlayerTag = "Player";
```

Comment:
Const identifiers should be in all caps:
public const string ENEMY_TAG
public const string TILEMAP_ROCK_TAG
etc

Reply:
Ok. I'll make the changes once I fix out the merge conflicts.